### PR TITLE
[admission-policy-engine] extend docs

### PR DESCRIPTION
**Pull request by yalosev**
Signed-off-by: Yuriy Losev <yuriy.losev@flant.com>

## Description
Docs about pod security standards extenstion

## Why do we need it, and what problem does it solve?
Users want to have a flow to extend current security standards. Provide the documentation for this action

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: admission-policy-engine
type: chore
summary: Describe how to add you own security pod policy to the existing onces.
impact_level: low
```

### CODE REVIEW

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/test'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/test'...

**Review by z9r5 — DISMISSED** [review]

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/test'...

File: modules/015-admission-policy-engine/docs/FAQ_RU.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+---
+
+## Как расширить политики Pod Security Standards?
+
+> Pod Security Standards реагируют на label `security.deckhouse.io/pod-policy: restricted` или `security.deckhouse.io/pod-policy: baseline`.
+
+Чтобы расширить политику Pod Security Standards, добавив к существующим проверкам политики свои собственные, необходимо:
+- Описать параметры проверки.
+- Привязать её к политике `restricted` или `baseline`.
+
```
Comment (z9r5) [inline modules/015-admission-policy-engine/docs/FAQ_RU.md:12]:
```suggestion
Чтобы расширить политику Pod Security Standards, добавив к существующим проверкам политики свои собственные, необходимо:
- Создать шаблон проверки.
- Привязать его к политике `restricted` или `baseline`.

```

File: modules/015-admission-policy-engine/docs/FAQ_RU.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+## Как расширить политики Pod Security Standards?
+
+> Pod Security Standards реагируют на label `security.deckhouse.io/pod-policy: restricted` или `security.deckhouse.io/pod-policy: baseline`.
+
+Чтобы расширить политику Pod Security Standards, добавив к существующим проверкам политики свои собственные, необходимо:
+- Создать шаблон проверки.
+- Привязать его к политике `restricted` или `baseline`.
+
+Пример описания параметров проверки адреса репозитория образа контейнера:
```
Comment (z9r5) [inline modules/015-admission-policy-engine/docs/FAQ_RU.md:13]:
```suggestion
Пример шаблона для проверки адреса репозитория образа контейнера:
```

File: modules/015-admission-policy-engine/docs/FAQ.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+
+## How to extend Pod Security Standards policies?
+
+> Pod Security Standards respond to the `security.deckhouse.io/pod-policy: restricted` or `security.deckhouse.io/pod-policy: baseline` label.
+
+To extend the Pod Security Standards policy by adding your checks to existing checks, you need to:
+- Describe the parameters of the check.
+- Bind it to the `restricted` or `baseline` policy.
+
+Example of the description of the parameters for checking the URL of the container image repository:
```
Comment (z9r5) [inline modules/015-admission-policy-engine/docs/FAQ.md:13]:
```suggestion
To extend the Pod Security Standards policy by adding your checks to existing checks, you need to:
- Create a constraint template for the check (ресурс `ConstraintTemplate`).
- Bind it to the `restricted` or `baseline` policy.

Example of the `ConstraintTemplate` for checking a  repository URL of a container image:
```

File: modules/015-admission-policy-engine/docs/FAQ_RU.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
+---
+title: "Модуль admission-policy-engine: FAQ"
+---
+
+## Как расширить политики Pod Security Standards?
+
+> Pod Security Standards реагируют на label `security.deckhouse.io/pod-policy: restricted` или `security.deckhouse.io/pod-policy: baseline`.
+
+Чтобы расширить политику Pod Security Standards, добавив к существующим проверкам политики свои собственные, необходимо:
+- Создать шаблон проверки.
```
Comment (z9r5) [inline modules/015-admission-policy-engine/docs/FAQ_RU.md:10]:
```suggestion
- Создать шаблон проверки (ресурс `ConstraintTemplate`).
```

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/test'...

**Review by z9r5 — APPROVED** [review]

**Comment by deckhouse-BOaTswain** [comment]
Cherry pick PR [2889](https://github.com/deckhouse/deckhouse/pull/2889) to the branch [release-1.38](https://github.com/deckhouse/deckhouse/tree/release-1.38) successful!

**Pull request merged**